Mat contact changes

mathewmorris opened the pull request:
resolves #70

Commit:
Changed up contact page so we can get rid of feedback page. Added icons for contact page.

smurphy8 reviewed (commented):
It would be helpful to throw some screen shots of what these changes look like into the comments on this PR @mathewmorris it looks like good work though!

mathewmorris commented:
Before:
![image](https://cloud.githubusercontent.com/assets/10860014/20015790/d78851de-a28a-11e6-98ee-4b7a6557215a.png)
After:
![image](https://cloud.githubusercontent.com/assets/10860014/20015798/ded324be-a28a-11e6-970f-55bb1013816e.png)

mathewmorris commented:
@smurphy8 Also, should I be creating new branches so that you can merge it in? Or should I merge into my master branch and then create a PR?

Sorry, I'm a newbie at this PR stuff 😆

smurphy8 commented:
Do it just like this, new branch on your side ... pointed at master

On Nov 4, 2016 12:36 PM, "Mat Morris" notifications@github.com wrote:

> @smurphy8 https://github.com/smurphy8 Also, should I be creating new
> branches so that you can merge it in? Or should I merge into my master
> branch and then create a PR?
> 
> Sorry, I'm a newbie at this PR stuff 😆
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/codeforokc/openbudgetokc/pull/77#issuecomment-258498100,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AALJjJO1NN8OlycSemH8wMohxCt3OXwoks5q620CgaJpZM4KmuBj
> .

mkchandler commented on _src/contact.jade:
```diff
@@ -4,14 +4,33 @@
-          a(href='https://www.facebook.com/CodeForOKC/')
-            | facebook.com/CodeForOKC
+        div
+          h2.subheader How would you use Open Budget?
```
@mathewmorris can you adjust this to match the title of the site used everywhere else? "Open Budget: OKC" instead of just "Open Budget"

mathewmorris replied:
Yeah! 👍  I gotcha bro. haha

Commit:
Changed 'Open Budget' to 'Open Budget: OKC' to match.

mathewmorris commented:
So, it says there is a conflict with `_src/what-we-do.jade`. The only thing I changed on there is the link on the bottom. It was pointing to `feedback.html` but I changed it to `contact.html`. What do I need to do to resolve the conflict?

smurphy8 commented:
@mathewmorris if contact.html is correct then find the points where the conflict starts and end in the file.
https://githowto.com/resolving_conflicts

mathewmorris commented:
@smurphy8 The place I'm seeing the conflict is only on this PR. There's no conflict on my side.

smurphy8 commented:
Yeah it is messy.  I  am working on it

On Nov 6, 2016 10:14 PM, "Mat Morris" notifications@github.com wrote:

> @smurphy8 https://github.com/smurphy8 The place I'm seeing the conflict
> is only on this PR. There's no conflict on my side.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/codeforokc/openbudgetokc/pull/77#issuecomment-258743830,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AALJjN0SgTEFWe3EWcO8T8WfFTOAQqykks5q7qWigaJpZM4KmuBj
> .

smurphy8 commented:
Oh whoops wrong pr

On Nov 7, 2016 8:42 AM, "Scott Murphy" scottmurphy09@gmail.com wrote:

> Yeah it is messy.  I  am working on it
> 
> On Nov 6, 2016 10:14 PM, "Mat Morris" notifications@github.com wrote:
> 
> > @smurphy8 https://github.com/smurphy8 The place I'm seeing the
> > conflict is only on this PR. There's no conflict on my side.
> > 
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > https://github.com/codeforokc/openbudgetokc/pull/77#issuecomment-258743830,
> > or mute the thread
> > https://github.com/notifications/unsubscribe-auth/AALJjN0SgTEFWe3EWcO8T8WfFTOAQqykks5q7qWigaJpZM4KmuBj
> > .

Commit:
Changed what-we-do.html link from feedback.html to contact.html.

jagthedrummer reviewed (approved):
Looks good to me!

mathewmorris commented:
Woo!!! I love it! haha
